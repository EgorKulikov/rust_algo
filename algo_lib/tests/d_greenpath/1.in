5 12
1 2
1 3
2 4
2 5
1 1 1
2 1
2 2
2 3
2 4
2 5
1 2 2
2 1
2 2
2 3
2 4
2 5
