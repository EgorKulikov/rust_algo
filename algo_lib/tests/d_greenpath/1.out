1
199648871
399297742
199648871
199648871
598946614
199648873
2
2
2
